apps/emqx/mix.exs

```diff
@@ -35,7 +35,7 @@ defmodule EMQX.MixProject do
     [
       %{
         src: "./src/emqx_persistent_session_ds/DurableSession.asn",
-        compile_opts: [:per, :noobj, :no_ok_wrapper, outdir: ~c"gen_src"]
```

Comment:
A/B test results:

Median session size in 1:1 test with 100k clients
PER: 1473
BER: 1720

Throughput test: https://snapshots.raintank.io/dashboard/snapshot/5gbFrecYJAEe8ODe22ubGRy0AC2ah5gC

No significant difference.